### DISCUSSION TITLE
SSR cache?

### DISCUSSION THREAD

**Discussion by xblabs** [inertiajs/inertia#920] [Help]
New to inertia. I read about the SSR feature in beta and was wondering if that also allows for caching results to redis or static files for other services to obtain from the cache if needed. Does anyone know?

**Comment by reinink** [comment]
So I haven't personally implemented any caching yet, because it's quite fast, but you absolutely could implement this yourself. Right now the SSR code in early access is very low level (basically no abstractions), so it's totally possible to take the results of the first SSR hit and cache them somewhere, and then use that cache for future requests.

**Comment by xblabs** [comment]
Thanks Jonathan! That means there might or might not be an implementation later on / out of beta?  In case of a custom implementation, will it be as simple as having ideally a singular interceptor point in the sequence, i.e. the SSR render end result?  Off topic : your Laravel Eloquent course is fire!

**Comment by craigrileyuk** [comment]
Now that Inertia 1.0.0 is out, I was hoping that this is something that could be revisited.

Obviously the SSR process is pretty quick, but if you're getting hundreds of requests per minute for pages, obviously it's going to put a bit of a tax on resources that aren't completely necessary.

Now that the SSR rendering processes have been abstracted away from userland, it would be cool if you could pass the SSR server a cache setup somehow for using something like Redis.
